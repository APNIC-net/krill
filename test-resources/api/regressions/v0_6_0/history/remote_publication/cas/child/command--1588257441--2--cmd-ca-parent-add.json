{
  "actor": "krill",
  "time": "2020-04-30T14:37:21.262637Z",
  "handle": "child",
  "version": 2,
  "sequence": 2,
  "details": {
    "add_parent": [
      "ta",
      "embedded"
    ]
  },
  "effect": {
    "events": [
      2
    ]
  }
}