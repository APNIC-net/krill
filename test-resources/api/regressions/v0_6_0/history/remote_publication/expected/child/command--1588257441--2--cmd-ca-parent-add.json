{
  "command": {
    "actor": "krill",
    "time": "2020-04-30T14:37:21.262637Z",
    "handle": "child",
    "version": 2,
    "sequence": 2,
    "details": {
      "add_parent": [
        "ta",
        "embedded"
      ]
    },
    "effect": {
      "events": [
        2
      ]
    }
  },
  "result": {
    "Events": [
      {
        "id": "child",
        "version": 2,
        "details": {
          "parent_added": [
            "ta",
            "embedded"
          ]
        }
      }
    ]
  }
}